### ISSUE TITLE
space between UFEAT and value

### ISSUE THREAD

**Issue by vistamou** [Arborator/arborator-frontend#121]
Hello, 

Recently I've tried to export the conll annotated file and I observed that 
between the FEAT and the value there is a **space** (that normally it shouldn't be there).

here's an example:

`Case= Acc|Definite= Def|Gender= Neut|Number= Sing|PronType= Art`

could you please check this?

Thank you :)

**Comment by bguil** [comment]
This is not the case with other treebanks. Can you check the original data you use to start the treebank is correctly formatted?

If possible, give us an CoNLL tree example from this data to help up test.

**Comment by vistamou** [comment]
ok yes, you're right something it's causing this, because for example I saw another project (https://arboratorgrew.elizia.net/#/projects/SUD_Breton_KEB) and it's fine when exporting the data--

and also I corrected my data and uploaded new version of them and still I could download without problems-- 

but I'm wondering what could be wrong in the previous process? could it be some mistake in the metadata file? which might have introduce the space between FEATS?
